### ISSUE TITLE
HTTP_PROXY not used for video playback

### ISSUE THREAD

**Issue by Brueggus** [flaviotordini/minitube#23]
I've set HTTP_PROXY and HTTPS_PROXY environment variables to make minitube use a proxy server. When I play a video, `netstat -Wpc | grep minitube` shows:
```
tcp        0      0 thinkpad:33572          my.proxy.com:21518 ESTABLISHED 9693/minitube       
tcp        0      0 thinkpad:33569          my.proxy.com:21518 ESTABLISHED 9693/minitube       
tcp        0      0 thinkpad:33574          my.proxy.com:21518 ESTABLISHED 9693/minitube       
tcp        0      0 thinkpad:33570          my.proxy.com:21518 ESTABLISHED 9693/minitube       
tcp6       0      0 pNothingToSeeHere.dip0.t-ipconnect.de:37432 2a00:1450:4005:3::a:https ESTABLISHED 9693/minitube   
```

minitube seems to use the proxy server to grab everything but the video itself. The last line shows that it connects directly to Google/Youtube.

Could IPv6 or HTTPS be an issue?

**Comment by Brueggus** [comment]
This issue seems to be fixed in minitube 2.5.

**Comment by flaviotordini** [comment]
This must be because 2.5 is packaged to use gstreamer instead of VLC. Probably the VLC backend does not support the proxy setting.
